src/include/common/types/types.h

```diff
@@ -224,6 +224,7 @@ enum class PhysicalTypeID : uint8_t {
     INTERNAL_ID = 14,
     ALP_EXCEPTION_FLOAT = 15,
     ALP_EXCEPTION_DOUBLE = 16,
+    DECIMAL = 17,
```

Comment:
As always, to implement an API binding, not a single line from C++ side should be changed!